ionoscloud/resource_k8s_cluster_test.go

```diff
@@ -144,6 +142,56 @@ func TestAccK8sClusterPrivate(t *testing.T) {
 	})
 }
 
+func TestAccK8sClusters(t *testing.T) {
+	var k8sCluster ionoscloud.KubernetesCluster
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckK8sClusterDestroyCheck,
+		Steps: []resource.TestStep{
```

Comment:
maybe add an error case, where the filters don't match anything

Reply:
added a no match case ([f6d84ca](https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/519/commits/f6d84ca77aa5f5db67d18e25d54353ec8a480c4a))